Parse/ParseClientConfiguration.h

```diff
@@ -10,6 +10,7 @@
 #import <Foundation/Foundation.h>
 
 #import <Parse/PFConstants.h>
+#import "PFFileUploadController.h"
```

Comment:
import should probably be `#import <Parse/PFFileUploadController.h>` or even better, a forward `@class PFFileUploadController`